code/modules/economy/shippingmarket.dm

```diff
@@ -282,6 +282,35 @@
 
 		shipped_thing.throw_at(target, 100, 1)
 
+	proc/clear_path_to_market()
+		var/turf/T
+		var/list/bounds = get_area_turfs(/area/supply/delivery_point)
+		bounds += get_area_turfs(/area/supply/sell_point)
+		bounds += get_area_turfs(/area/supply/spawn_point)
+		var/min_x = INFINITY
+		var/max_x = 0
+		var/min_y = INFINITY
+		var/max_y = 0
+		for(T in bounds)
```

Comment:
why are you reusing variables :waa:

Reply:
T#1 T#2, often leads to confusion on my part and "Naming Things: Hardest problem in programming'


But yeah I can uniquely identify the sets.